migrations/20240414170023_create_userProfile_table.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable("userProfile", (table) => {
```

Comment:
It's better to change the table name to snake case and use the plural form: 'user_profiles'.